From 30742c681eccafd2a47fa705b47ce730ffe178d8 Mon Sep 17 00:00:00 2001
From: Grische <10663446-grische@users.noreply.gitlab.com>
Date: Fri, 3 Nov 2023 00:27:40 +0100
Subject: [PATCH] targets: drop TP-Link RE305

The flash size is not sufficient to host updated packages
---
 targets/ramips-mt76x8 | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/targets/ramips-mt76x8 b/targets/ramips-mt76x8
index 52fb71be..1e27ac35 100644
--- a/targets/ramips-mt76x8
+++ b/targets/ramips-mt76x8
@@ -62,10 +62,6 @@ device('tp-link-re200-v2', 'tplink_re200-v2')
 
 device('tp-link-re200-v3', 'tplink_re200-v3')
 
-device('tp-link-re305', 'tplink_re305-v1', {
-	class = 'tiny', -- Only 6M of usable Firmware space
-})
-
 device('tp-link-tl-mr3020-v3', 'tplink_tl-mr3020-v3', {
 	factory = false,
 	extra_images = {
-- 
2.25.1

